lib/hawk/model/association.rb

```diff
@@ -73,17 +73,16 @@ module ClassMethods
         def inherited(subclass)
           super
 
-          parent = self
           subclass.instance_eval do
             # Inherit associations
             @_associations ||= {}
 
-            parent.associations.each do |name, (type, options)|
```

Comment:
As discussed verbally, here the `parent` class is saved in a closure in order to copy the associations from the parent in the inherited class. If you remove it, then ..... ------>